tester_app/config/initializers/react_on_rails.rb

```diff
@@ -0,0 +1,21 @@
+# Shown below are the defaults for configuration
+ReactOnRails.configure do |config|
```

Comment:
# which means that if you don't specify an option for the render_component helper, you'll get these options.